lib/streamlit/elements/widgets/button_group.py

```diff
@@ -181,57 +181,85 @@ def feedback(
     ) -> int | None:
         """Display a feedback widget.
 
-        This is useful to collect user feedback, especially in chat-based apps.
+        A feedback widget is an icon-based button group available in three
+        styles, as described in ``options``. It is commonly used in chat and AI
+        apps to allow users to rate responses.
 
-        Parameters:
-        -----------
+        Parameters
+        ----------
         options: "thumbs", "faces", or "stars"
-            The feedback options displayed to the user. The options are:
-            - "thumbs" (default): displays a row of thumbs-up and thumbs-down buttons.
-            - "faces": displays a row of five buttons with facial expressions, each
-                depicting increasing satisfaction from left to right.
-            - "stars": displays a row of star icons typically used for ratings.
+            The feedback options displayed to the user. ``options`` can be one
+            of the following:
+
+            - ``"thumbs"`` (default): Streamlit displays a thumb-up and
```

Comment:
nit: In the `options` docstrings, `Streamlit displays` can be shortened to `displays`. We don't prepend Streamlit to every line in docstrings for other elements.

Reply:
This is a newer direction I'm going to use more complete sentences in docstrings (see `on_select` and `selection_mode` for [`st.plotly_chart`](https://docs.streamlit.io/develop/api-reference/charts/st.plotly_chart). Since bullet points in this form (value: description) can be more complicated and have more than one sentence to describe each one, I'm leaning toward complete sentences for broader consistency (plus good for non-native English speakers, translations, and AI-consumption). :)